src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -8,53 +8,47 @@
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
+
 import java.util.Properties;
-import java.io.File;
 import java.util.Random;
 
-
 /**
- *
  * This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
  * After successful connection, it puts random entries into the map.
- *
- * See: https://docs.hazelcast.cloud/docs/java-client
- *
+ * <p>
+ * See: <a href="https://docs.cloud.hazelcast.com/docs/java-client">https://docs.cloud.hazelcast.com/docs/java-client</a>
  */
 public class ClientWithSsl {
 
-    public static void main(String[] args) throws InterruptedException {
-      ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
-      Properties props = new Properties();
-      props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").getPath());
-      props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_SSL_PASSWORD");
-      props.setProperty("javax.net.ssl.trustStore", classLoader.getResource("client.truststore").getPath());
-      props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_SSL_PASSWORD");
-      ClientConfig config = new ClientConfig();
-      config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
-      config.setGroupConfig(new GroupConfig("YOUR_CLUSTER_NAME", "YOUR_CLUSTER_PASSWORD"));
-      config.setProperty("hazelcast.client.statistics.enabled","true");
-      config.setProperty(ClientProperty.HAZELCAST_CLOUD_DISCOVERY_TOKEN.getName(), "YOUR_CLUSTER_DISCOVERY_TOKEN");
-      config.setProperty(HazelcastCloudDiscovery.CLOUD_URL_BASE_PROPERTY.getName(), "YOUR_DISCOVERY_URL");
-      HazelcastInstance client = HazelcastClient.newHazelcastClient(config);
-      IMap<String, String> map = client.getMap("map");
-      map.put("key", "value");
-      if(map.get("key").equals("value")) {
+    public static void main(String[] args) {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+        Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").getPath());
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_SSL_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore", classLoader.getResource("client.truststore").getPath());
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_SSL_PASSWORD");
+
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.setGroupConfig(new GroupConfig("YOUR_CLUSTER_NAME", "YOUR_CLUSTER_PASSWORD"));
+        config.setProperty("hazelcast.client.statistics.enabled", "true");
+        config.setProperty(ClientProperty.HAZELCAST_CLOUD_DISCOVERY_TOKEN.getName(), "YOUR_CLUSTER_DISCOVERY_TOKEN");
+        config.setProperty(HazelcastCloudDiscovery.CLOUD_URL_BASE_PROPERTY.getName(), "YOUR_DISCOVERY_URL");
```

Comment:
is this actually necessary? isn't the URL always the same?

Reply:
@jerrinot we are using same logic on our non prod environments, so we override the default value.

Reply:
@huseyinbabal I think we can remove this to prevent any user confusion. And if the env is not production you can add this line while generating the package. wdyt?

Reply:
It's removed in the PR